pkg/roachprod/opentelemetry/files/opentelemetry-config.yaml.tmpl

```diff
@@ -0,0 +1,122 @@
+---
+extensions:
+  health_check:
+
+receivers:
+  {{- if .CockroachDBMetrics }}
+  prometheus/cockroachdb:
+    config:
+      global:
+        scrape_interval: 30s
+      scrape_configs:
+      - job_name: cockroachdb
+        honor_timestamps: false
+        metrics_path: /_status/vars
+        tls_config:
+          insecure_skip_verify: true
+        follow_redirects: true
+        enable_http2: true
+        static_configs:
+          # Multiple targets are listed to cover the cases when the default
+          # HTTP port is used or when scanning for a free HTTP port starting
+          # from 29000. In the latter case 29000 will be used for SQL and 29001
+          # for HTTP.
+          - targets:
+            - "localhost:26258"
+            - "localhost:29001"
+  {{- end }}
+  hostmetrics:
+    collection_interval: 10s
+    scrapers:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      load:
+      memory:
+      network:
+      processes:
+
+processors:
```

Comment:
Since we wind up talking about this a lot: are we setting the datadog host tag implicitly? What will the host tags look like for metrics sent with otel vs logs sent with fluent-bit?

Reply:
In `pkg/roachprod/opentelemetry/opentelemetry.go` we're setting the `host` tag via `fmt.Sprintf("host:%s", vm.Name(c.Name, int(node))),`. This is not a resource attribute but rather a data point attribute. Perhaps I should set `datadog.host.name` to be safe. What do you think?

Both Fluent Bit and OpenTelemetry are using `vm.Name(c.Name, int(node))` as their hostname. I'll spin up a new cluster to test.